Plugin.cs

```diff
@@ -153,15 +148,71 @@ private void OnNetworkEvent(IntPtr dataPtr, ushort opCode, uint sourceActorId, u
             if (!this.configuration.alwaysOn && (!this.configuration.smart || !Retainer())) return;
             var listing = MarketBoardCurrentOfferings.Read(dataPtr);
             var i = 0;
-            if (this.configuration.hq && this.items.Single(j => j.RowId == listing.ItemListings[0].CatalogId).CanBeHq)
-            {
```

Comment:
This should be the only line that uses this.items, so by removing it you should be able to get rid of the Lumina.Excel.GeneratedSheets dependency.

Reply:
Wasn't sure what this.items refers to..
Does this refers to the item we want to sell?

Reply:
I mean the `this.items.Single(j => j.RowId == listing.ItemListings[0].CatalogId).CanBeHq)` is the only time we touch `this.items`, so you can get rid of the Excel sheet if we're not using it anymore.